README.md

```diff
@@ -241,7 +305,7 @@ Set these to match your settings:
 UPBOUND_ORG=acme
 UPBOUND_ACCOUNT_EMAIL=me@acme.io
 REPO=platform-ref-aws
-VERSION_TAG=v0.1.4
+VERSION_TAG=v0.2.0
 REGISTRY=registry.upbound.io
 PLATFORM_CONFIG=${REGISTRY:+$REGISTRY/}${UPBOUND_ORG}/${REPO}:${VERSION_TAG}
 ```
```

Comment:
@stevendborrelli can we update the `build` and `push` commands to use `up` also? :)

Reply:
Yes, I'll update that line.